public/netsuke/assets/js/docs-scrollspy.js

```
@@ -0,0 +1,91 @@
+        }
+
+        function update() {
+            var atBottom =
+                window.innerHeight + window.scrollY >=
+                document.documentElement.scrollHeight - 2;
+            if (atBottom) {
+                setActive(targets[targets.length - 1]);
+                return;
+            }
+            var active = null;
+            for (var i = 0; i < targets.length; i += 1) {
```

Biome (2.5.5)

[error] This var should be declared at the root of the enclosing function.

(lint/correctness/noInnerDeclarations)